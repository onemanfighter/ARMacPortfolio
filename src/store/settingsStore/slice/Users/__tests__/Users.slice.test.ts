import { renderHook } from '@testing-library/react-hooks';
import { settingsStore } from '@settingsStore';

describe('Users slice', () => {
  it('should return default terminal state', () => {
    const { result } = renderHook(() => settingsStore());

    expect(result.current.Users).toMatchSnapshot();
  });
});
