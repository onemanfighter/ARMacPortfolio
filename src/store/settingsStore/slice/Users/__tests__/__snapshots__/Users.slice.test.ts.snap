// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users slice should return default terminal state 1`] = `
{
  "amitraikwar": {
    "createdAt": "2024-05-13",
    "email": "raikwar.amit.1603@gmail.com",
    "id": "amitraikwar",
    "name": "Amit Raikwar",
    "password": "Amit",
    "profilePicture": "https://avatars.githubusercontent.com/u/104697219?s=400&u=7ecc539c268755cfe0409ca5863773bb726387ba&v=4",
    "role": "admin",
    "updatedAt": "2021-09-02",
  },
}
`;
